packages/fuels-signers/src/wallet.rs

```diff
@@ -288,13 +290,58 @@ impl Wallet {
         ];
 
         // Build transaction and sign it
-        let mut tx =
-            self.get_provider()
-                .unwrap()
-                .build_transfer_tx(&inputs, &outputs, tx_parameters);
-        let _sig = self.sign_transaction(&mut tx).await.unwrap();
+        let mut tx = self
+            .get_provider()?
+            .build_transfer_tx(&inputs, &outputs, tx_parameters);
+        let _sig = self.sign_transaction(&mut tx).await?;
 
-        let receipts = self.get_provider().unwrap().send_transaction(&tx).await?;
+        let receipts = self.get_provider()?.send_transaction(&tx).await?;
+
+        Ok((tx.id().to_string(), receipts))
+    }
+
+    /// UNCONDITIONAL transfer of `amount` coins of type `asset_id` to
+    /// the contract at `to`.
+    /// Fails if amount for asset ID is larger than this wallets spendable coins.
+    /// Returns the transaction ID that was sent and the list of receipts.
+    ///
+    /// CAUTION !!!
+    ///
+    /// This will transfer coins to a contract even with no way to retrieve them
+    /// (i.e. no withdrawal functionality on receiving contract), possibly leading
+    /// to the PERMANENT LOSS OF COINS if not used with care.
```

Comment:
```suggestion
    /// Unconditionally transfers `amount` coins of type `asset_id` to
    /// the contract at `to`.
    /// Fails if amount for `asset_id` is larger than this wallet's spendable coins.
    /// Returns the corresponding transaction ID and the list of receipts.
    ///
    /// CAUTION !!!
    ///
    /// This will transfer coins to a contract, possibly leading
    /// to the PERMANENT LOSS OF COINS if not used with care.
```